ecommerce/coupons/tests/mixins.py

```diff
@@ -271,35 +279,37 @@ def mock_course_runs_contains_endpoint_failure(self, course_run_ids, catalog_id,
         Helper function to register a discovery API endpoint with failure
         for getting course runs information.
         """
-        def callback(request, uri, headers):  # pylint: disable=unused-argument
-            raise error
-
         catalog_contains_course_run_url = '{}catalogs/{}/contains/?course_run_id={}'.format(
             discovery_api_url,
             catalog_id,
             ','.join(course_run_id for course_run_id in course_run_ids),
         )
-        httpretty.register_uri(
-            method=httpretty.GET,
-            uri=catalog_contains_course_run_url,
-            responses=[
-                httpretty.Response(body=callback, content_type='application/json', status_code=500)
-            ]
+        responses.add(
+            method=responses.GET,
+            url=catalog_contains_course_run_url,
+            body=error(),
+            content_type='application/json',
+            status=500
         )
 
-    def mock_catalog_query_contains_endpoint(self, course_run_ids, course_uuids, absent_ids, query, discovery_api_url):
+    def mock_catalog_query_contains_endpoint(
+        self, course_run_ids, course_uuids, absent_ids, query, discovery_api_url, partner='edx'
+    ):
         query_contains_info = {str(identifier): True for identifier in course_run_ids + course_uuids}
         for identifier in absent_ids:
             query_contains_info[str(identifier)] = False
         query_contains_info_json = json.dumps(query_contains_info)
-        url = '{base}catalog/query_contains/?course_run_ids={run_ids}&course_uuids={uuids}&query={query}'.format(
+        url = (
+            '{base}catalog/query_contains/?course_run_ids={run_ids}&course_uuids={uuids}&query={query}&partner={prtnr}'
+        ).format(
             base=discovery_api_url,
             run_ids=",".join(course_run_id for course_run_id in course_run_ids),
             uuids=",".join(str(course_uuid) for course_uuid in course_uuids),
-            query=query
-        )
-        httpretty.register_uri(
-            httpretty.GET, url,
+            query=query,
+            prtnr=partner
```

Comment:
Are you sure `prtnr` is a good naming for the parameter, won't partner be better? )))